quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/FeaturePropertyMappers.java

```diff
@@ -27,10 +31,64 @@ public List<PropertyMapper<?>> getPropertyMappers() {
                         .build(),
                 fromOption(FeatureOptions.FEATURES_DISABLED)
                         .paramLabel("feature")
+                        .build(),
+                fromOption(FeatureOptions.FEATURE)
+                        .paramLabel("enabled|disabled|vX(X is version)")
+                        .wildcardKeysValidator(FeaturePropertyMappers::validateSingleFeature)
                         .build()
         );
     }
 
+    @Override
+    public void validateConfig(Picocli picocli) {
+        validateSingleFeatureDuplicates(picocli);
+    }
+
+    // Verify a feature specified via '--feature-<name>' is set only once on CLI
+    private static void validateSingleFeatureDuplicates(Picocli picocli) {
```

Comment:
This should be removed. If we want to validate this, then it should be done generally under https://github.com/keycloak/keycloak/issues/43604

It's straight-forward to do this in the relevant config sources if we want to go that route.

Reply:
Ok, works to me to do it generally.